templates/blog/meta.ts

```diff
@@ -0,0 +1,65 @@
+// Generated using "yarn build-templates"
+
+export const meta = {
+  name: "Blog",
+  description:
+    "This is a simple self-hosted, lightweight, singe-user PHP blog, where you can create your own Facebook-like feed. Give read access to other people, and you can share rich text with photos including highlighted code or links.",
+  instructions: null,
+  changeLog: [{ date: "2022-12-21", description: "first release" }],
+  links: [{ label: "Github", url: "https://github.com/m1k1o/blog" }],
+  contributors: [
+    { name: "Supernova3339", url: "https://github.com/Supernova3339" },
+  ],
+  schema: {
+    type: "object",
+    required: [
+      "projectName",
+      "appServiceName",
+      "appServiceImage",
+      "blogTitle",
+      "blogName",
+      "blogNick",
+      "blogPass",
+    ],
+    properties: {
+      projectName: { type: "string", title: "Project Name" },
+      appServiceName: {
+        type: "string",
+        title: "App Service Name",
+        default: "blog",
+      },
+      appServiceImage: {
+        type: "string",
+        title: "App Service Image",
+        default: "m1k1o/blog:latest",
+      },
+      blogTitle: { type: "string", title: "Blog Title" },
+      blogName: { type: "string", title: "Blog Name" },
+      blogNick: { type: "string", title: "Username" },
+      blogPass: { type: "string", title: "Password" },
+      blogLang: { type: "string", title: "Language", default: "EN" },
+    },
+  },
+  logo: "logo.png",
```

Comment:
The logo is the FB favicon from what I can see. If it doesn't have a real one, just remove it and a placeholder will be used by Easypanel.

Reply:
This is the official logo supplied in the github repo, hence what I used.